### PR TITLE
[front] enh: Add support for array in inputs, improved browse result and details

### PR DESCRIPTION
**Pull request by tdraier**
## Description

- Improved app return and browse details view
- use array of string as input
- add support for arrays in multi-actions-agent

## Deploy Plan

Deploy front

### CODE REVIEW

File: types/src/front/assistant/actions/dust_app_run.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export type DustAppRunConfigurationType = {
 };
 
 export type DustAppParameters = {
-  [key: string]: string | number | boolean;
+  [key: string]: string | number | boolean | string[] | number[] | boolean[];
```
Comment (spolu) [inline types/src/front/assistant/actions/dust_app_run.ts:18]: Hello! That's not true in the context of DustAppRun which only supports string/number/boolean

Why do you need this?

**Review by spolu — COMMENTED** [review]
Looks good but don't understand the Dust App run diff?

**Comment by tdraier** [comment]
> Looks good but don't understand the Dust App run diff?

One change on the browse app to change the output schema, the other change is on the multi action app to support the array types ( see earlier discussion )

File: types/src/front/assistant/actions/dust_app_run.ts (hunk 1 above)
Comment (tdraier) [reply types/src/front/assistant/actions/dust_app_run.ts:18]: The idea was actually to add support for arrays in dust app parameters - as the "urls" for the [browse app](https://dust.tt/w/78bda07b39/a/21092925b9). Hence, the change in [multi-action app](https://dust.tt/w/78bda07b39/a/0e9889c787) to send the json schema to the model (with the missing "items" property), which correctly generates string[] inputs when requested to do so. The app is receiving well formed json string array.

File: front/lib/api/assistant/actions/browse.ts [hunk 2]
```diff
@@ -108,12 +100,15 @@ export class BrowseConfigurationServerRunner extends BaseActionConfigurationServ
...
     return new Ok({
       name: name ?? "browse",
-      description: description ?? "Get content of a page.",
+      description: description ?? "Get the content of a web page.",
       inputs: [
         {
           name: "urls",
-          description: "List of urls to fetch, as a JSON array.",
-          type: "string",
+          description: "List of urls to fetch.",
```
Comment (spolu) [inline front/lib/api/assistant/actions/browse.ts:107]: I would use `browse` rather than `fetch` ?

**Review by spolu — APPROVED** [review]

**Pull request merged**